### ISSUE TITLE
Make centroid property of spectrum and not scan

### ISSUE THREAD

**Issue by trishorts** [smith-chem-wisc/mzLib#499]
![image](https://user-images.githubusercontent.com/16841846/61389520-b1921f80-a87e-11e9-9372-1bbb734533fc.png)

**Comment by zrolfs** [comment]
I disagree. The spectrum is simply the array of peaks. All information surrounding the spectrum (RT, scan number, instrument, centroid/profile) should be in the MsDataScan. Why do you want this?

**Comment by trishorts** [comment]
because the centroiding actually gets done in mzspectrum. So, seems like once it gets centroided, the variable should be set. The scan just takes what it gets. No point setting the centroid variable there.

**Comment by nbollis** [comment]
> *This was generated by AI during triage.*

The centroid property correctly belongs on \\MsDataScan\\, not \\MzSpectrum\\.

\\MzSpectrum\\ is the raw data array (m/z-intensity pairs). \\MsDataScan\\ wraps that data with acquisition metadata: RT, scan number, polarity, and centroid/profile mode. Centroiding is a transformation applied to raw data — the *result* (centroided vs. profile) is metadata about *how* the data was acquired, not about the data itself. Keeping \\IsCentroid\\ on the scan preserves the clean separation: spectrum = data, scan = context.

This aligns with the established design in mzLib. See \\.out-of-scope/centroid-location-on-scan.md\\ for the full decision record.

Closing as wontfix.
